adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -672,7 +649,7 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord]) extends Serializabl
    *                            observations to.
    * @return Returns an RDD of recalibrated reads.
    */
-  def adamBQSR(
+  def bqsr(
```

Comment:
If we're renaming this, I might prefer `recalibateBaseQualities` over `bqsr`. Thoughts?